src/Util/log.cpp

```diff
@@ -7,8 +7,53 @@
 #include <vector>
 #include <Util/config.h>
 
+#ifdef _WIN64
+#include <Windows.h>
+#endif
+
+#include "log.h"
+
 namespace logging {
 std::vector<spdlog::sink_ptr> sinks;
+constexpr bool log_file_exceptions = true;  // disable it to disable logging
+
+void flush() { spdlog::details::registry::instance().flush_all(); }
+
+#ifdef _WIN64
+
+static LONG WINAPI exception_handler(PEXCEPTION_POINTERS pExp) noexcept {
+    const unsigned ec = pExp->ExceptionRecord->ExceptionCode;
```

Comment:
```suggestion
    const u32 ec = pExp->ExceptionRecord->ExceptionCode;
```
Best to use the common types everywhere